packages/material-ui/src/Popper/Popper.js

```diff
@@ -121,25 +121,43 @@ const Popper = React.forwardRef(function Popper(props, ref) {
       }
     }
 
-    const popper = new PopperJs(getAnchorEl(anchorEl), tooltipRef.current, {
+    let popperModifiers = [
+      {
+        name: 'preventOverflow',
+        options: {
+          altBoundary: disablePortal,
+        },
+      },
+      {
+        name: 'flip',
+        options: {
+          altBoundary: disablePortal,
+        },
+      },
+      {
+        name: 'onUpdate',
+        enabled: true,
+        phase: 'afterWrite',
+        fn({ state }) {
+          createChainedFunction(handlePopperUpdate, popperOptions.onUpdate)(state);
+        },
+      },
+    ];
+
+    if (modifiers != null) {
```

Comment:
Modifiers get merged in the library so this works. I didn't make this an array check because we'd probably want it to fail loudly.

Reply:
propTypes should take care of runtime warnings if it's not an array.